test/plugins/parse5-flatten-image.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
I'd love to eventually see this with snapshot tests instead.

Reply:
Definitely not part of this PR :)

Reply:
This was intended as "nitpick"/":heart:"